Makefile

```diff
@@ -1,7 +1,8 @@
-test:
```

Comment:
let's re-add `test` - the point of a `Makefile` is to have all commands organized here